rust/agama-dbus-server/src/locale.rs

```diff
@@ -1,62 +1,54 @@
+pub mod helpers;
```

Comment:
NP: I think this should be renamed as `L10n` because it configures more localization options (locale, keymap, timezone, UILocale). Anyway, it is something we can change later.